business-registry-dashboard/app/stores/affiliations.ts

```diff
@@ -17,7 +17,7 @@ export const useAffiliationsStore = defineStore('brd-affiliations-store', () =>
       type: '',
       status: ''
     },
-    loading: false,
+    loading: true,
```

Comment:
I figured this would be the smartest and cleanest way to do it with minimal code change. Now, the initial value for loading is true so it doesn't go from: Not Loading --> Loading --> Not Loading (finished).

We now go: Loading --> Not Loading (finished)

Reply:
Awesome!

Does "no affiliation records" still display after loading (if needed)?

Reply:
Yup, it does! This is an account with 0 items in the table:
![image](https://github.com/user-attachments/assets/1b57174d-885d-48c3-bedd-5341fae57bba)